src/utils/localizationBundle/index.js

```diff
@@ -32,8 +36,16 @@ export default {
   },
   
   getPublished: {
-    title: 'Get Published',
+    title: 'To Do',
     description: 'Work with your mentor to complete these items so you can get published.',
+    createInstructorAccount: 'Create instructor account',
+    joinSlack: 'Join egghead Slack',
+    claimLesson: 'Claim new lesson' ,
+    submitRoughDraft: 'Submit rough draft',
+    getGear: 'Get gear',
+    recordWithGear: 'Re-record with gear',
+    iterate: 'Iterate until approved',
+    publish: 'Publish lesson',
```

Comment:
Moved this hard-coded text I had missed from before into the localization bundle